### PR TITLE
(feat). Add openebs adopters testimonials

### PR DESCRIPTION
**Pull request by AVRahul**
Signed-off-by: AVRahul <a.v.rahul11@gmail.com>

Add some of openEBS adopter's testimonials.

**screenshots**
Screenshot from 2021-06-14 17-21-58
Screenshot from 2021-06-14 17-21-45

:

### CODE REVIEW

File: website/src/pages/Home/index.tsx [hunk 1]
```diff
@@ -16,18 +16,20 @@ import Newsletter from "../../components/Newsletter";
 import {IconButton} from "@material-ui/core";
 import Sponsor from "../../components/Sponsor";
 import Slider from "react-slick";
-import { adaptorTestimonials } from './adaptorTestimonials';
+// import { adaptorTestimonials } from './adaptorTestimonials';
```
Comment (Pallavi-PH) [inline website/src/pages/Home/index.tsx:19]: Can we remove this and adaptorTestimonials.ts file?

File: website/src/pages/Home/index.tsx [hunk 2]
```diff
@@ -41,6 +43,19 @@ const Home: React.FC = () => {
 
     // const [logoSlidesPerScreen, setLogoSlidesPerScreen] = useState<number>(6);
 
+        //function to fecth all the adoters testimonials
+    const fetchAdoptersTestimonials = async () => {
+        await fetch(adopterData).then((response) => {
+         if (response.ok) return response.text();
```
Comment (satyapriyamishra222) [inline website/src/pages/Home/index.tsx:49]: Pls reformat it properly with indentations and blocks ?

File: website/src/pages/Home/index.tsx [hunk 3]
```diff
@@ -41,6 +43,19 @@ const Home: React.FC = () => {
...
+         else return Promise.reject("could't fetch text correctly");
```
Comment (satyapriyamishra222) [inline website/src/pages/Home/index.tsx:49]: Raw text ?

**Review by Pallavi-PH — APPROVED** [review]
Lgtm

**Review by IsAmrish — APPROVED** [review]
lgtm

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**